src/features/dataset/state/datasetActions.ts

```diff
@@ -17,6 +17,12 @@ export function loadDataset(ref: QriRef): ApiActionThunk {
   }
 }
 
+export function loadDatasetVersionInfo(ref: QriRef): ApiActionThunk {
```

Comment:
as below, this method should be called `loadHeader`. It is only temporarily returning `VersionInfo`s